utils/ziputil/zip_test.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2018 the Service Broker Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ziputil
+
+import (
+	"archive/zip"
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/GoogleCloudPlatform/gcp-service-broker/utils/stream"
+)
+
+var exampleFile = []byte{
```

Comment:
Consider adding a comment describing the contents of the example file here.

Reply:
Preview may not be updating, I added this:

```go
// A zip file with a single file with the contents "Hello, world!":
// MODE        SIZE  NAME
// -rw-rw-rw-  13    path/to/my/file.txt

```